themes/custom/droopler_theme/scss/layout/_modifiers.scss

```diff
@@ -225,3 +227,55 @@ $stripe-background-color: rgba(0,0,0, 0.57) !default;
   }
 }
 
+.tile {
+  padding: $image-background-padding-y 0 $tile-title-padding-y 0;
+  position: sticky;
+  cursor: pointer;
+  transform: scale(1);
+  transition: 0.3s;
+
+  @include media-breakpoint-down(md) {
+    margin-bottom: $tiles-spacing;
+  }
+
+  &:before {
+    content: none;
+  }
+
+  &:hover {
+    transform: scale(1.01);
+    transition: 0.3s;
+  }
+
+  .image-background-container {
+    position: absolute;
+    height: 100%;
+    top: 0;
+    right: 0;
+    display: block;
+    z-index: -1;
+    &.background-properties {
+      background-size: cover;
+      background-position: center center;
+      background-repeat: no-repeat;
+    }
+    &.background-image {
+      z-index: -1 !important;
+    }
+  }
+
+  .paragraph {
+    h2 {
+      text-align: center;
+      color: $white;
```

Comment:
Let's use `$color-odysseus` here (hope I remember the name, please check in `_colors.scss` :))